core/templates/dev/head/base.html

```diff
@@ -117,11 +117,11 @@
               <nav class="navbar navbar-default oppia-navbar oppia-prevent-selection" role="navigation">
                 <div class="navbar-container">
                   <div class="navbar-header protractor-test-navbar-header pull-left">
-                    <a ng-click="openSidebar()" class="navbar-brand oppia-navbar-menu oppia-transition-200">
+                    <a ng-if="windowWidth < 768" ng-click="openSidebar()" class="navbar-brand oppia-navbar-menu oppia-transition-200">
```

Comment:
@giritheja just checking, did you see this one? Probably want to use isWindowNarrow() etc.